src/main/java/at/hannibal2/skyhanni/data/ItemClickData.kt

```diff
@@ -23,18 +23,18 @@ class ItemClickData {
         if (packet is C08PacketPlayerBlockPlacement) {
             if (packet.placedBlockDirection != 255) {
                 val position = packet.position.toLorenzVec()
-                BlockClickEvent(ClickType.RIGHT_CLICK, position, packet.stack).postAndCatch()
+                event.isCanceled = BlockClickEvent(ClickType.RIGHT_CLICK, position, packet.stack).postAndCatch()
             } else {
-                ItemClickEvent(InventoryUtils.getItemInHand(), ClickType.RIGHT_CLICK).postAndCatch()
+                event.isCanceled = ItemClickEvent(InventoryUtils.getItemInHand(), ClickType.RIGHT_CLICK).postAndCatch()
             }
         }
         if (packet is C07PacketPlayerDigging && packet.status == C07PacketPlayerDigging.Action.START_DESTROY_BLOCK) {
             val position = packet.position.toLorenzVec()
-            BlockClickEvent(ClickType.LEFT_CLICK, position, InventoryUtils.getItemInHand()).postAndCatch()
-            ItemClickEvent(InventoryUtils.getItemInHand(), ClickType.RIGHT_CLICK).postAndCatch()
+            event.isCanceled = BlockClickEvent(ClickType.LEFT_CLICK, position, InventoryUtils.getItemInHand()).postAndCatch() ||
+                ItemClickEvent(InventoryUtils.getItemInHand(), ClickType.RIGHT_CLICK).postAndCatch()
```

Comment:
Consider cancelling the second event instead of using `||` which completely shortcircuits the second event:
```suggestion
            val blockClickCancelled = BlockClickEvent(ClickType.LEFT_CLICK, position, InventoryUtils.getItemInHand()).postAndCatch()
            event.isCanceled = ItemClickEvent(InventoryUtils.getItemInHand(), ClickType.RIGHT_CLICK).also { it.setCancelled(blockClickCancelled) }.postAndCatch()
```

This way `@SubscribeEvent(receiveCancelled = true)` will still get called.